region/maridia/outer.json

```diff
@@ -2423,6 +2485,40 @@
                   "devNote": "FIXME: It is possible to get over the space pirate, damageless, but it is challenging, especially when keeping the Skulltera alive for other frozen strats."
                 }
               ]
+            },
+            {
+              "id": 7,
+              "strats": [
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Gravity",
+                    {"or": [
+                      {"and": [
+                        "h_canArtificialMorphIBJ",
+                        "canTrickyJump",
+                        "canBombHorizontally"
+                      ]},
+                      {"and": [
+                        "h_canArtificialMorphSpringBallBombJump",
+                        "h_canArtificialMorphSpringBallBombJump"
+                      ]}
+                    ]}
+                  ],
+                  "note": [
+                    "With spring ball, get over the first pirate then spring ball bomb boost up to the ledge on the right, spring ball onto the platform with the pirate then up and go as far right as possible.",
+                    "One final spring ball bomb boost is needed to get to the top right door. Note that the Pirate's hitbox is taller than it looks.",
+                    "Without spring ball, using IBJ is trickier. IBJ over the pirate while ensuring that it doesnt fire invisible stationary lasers too close to the right.",
+                    "Get over the pirate while it isn't too close to the left wall in order to have time to IBJ up before it touches Samus. Then IBJ multiple times to get to the top right door."
```

Comment:
I think I take the 2->5 path then kill the fish with bombs; go under the rightish pirate and IBJ up.